docs/decisions/0001-connexion-replacement.md

```diff
@@ -0,0 +1,602 @@
+# Connexion Alternatives Decision Record
```

Comment:
For your convenience, you can see the formatted document [here](https://github.com/navapbc/template-application-flask/blob/chouinar/connexion-replacement/docs/decisions/0001-connexion-replacement.md).